easybuild/toolchains/compiler/intel_compilers.py

```diff
@@ -69,6 +69,11 @@ def set_variables(self):
             self.COMPILER_F77 = 'ifx'
             self.COMPILER_F90 = 'ifx'
             self.COMPILER_FC = 'ifx'
+            self.COMPILER_MPICXX = 'mpiicpc -cxx=icpx'
```

Comment:
@hattom This is actually not needed, I think.

EasyBuild already sets `$I_MPI_CC` & co to control which compiler is used by the MPI wrapper commands like `mpiicc`, and that seems to be working fine:

```
$ mpiicc --version
icc: remark #10441: The Intel(R) C++ Compiler Classic (ICC) is deprecated and will be removed from product release in the second half of 2023. The Intel(R) oneAPI DPC++/C++ Compiler (ICX) is the recommended compiler moving forward. Please transition to use this compiler. Use '-diag-disable=10441' to disable this message.
icc (ICC) 2021.7.0 20220726
Copyright (C) 1985-2022 Intel Corporation.  All rights reserved.
```

```
$ I_MPI_CC=icx mpiicc --version
Intel(R) oneAPI DPC++/C++ Compiler 2022.2.0 (2022.2.0.20220730)
Target: x86_64-unknown-linux-gnu
Thread model: posix
InstalledDir: /software/intel-compilers/2022.2.0/compiler/2022.2.0/linux/bin-llvm
Configuration file: /software/intel-compilers/2022.2.0/compiler/2022.2.0/linux/bin/icx.cfg
```

Reply:
Looks good. Let's close this PR.